content/hardware/06.nicla/boards/nicla-vision/product.md

```diff
@@ -7,8 +7,9 @@ primary_button_title: User Manual
 secondary_button_url: /hardware/nicla-vision/#tutorials
 secondary_button_title: All Tutorials
 core: arduino:mbed_nicla
-certifications: [CE, UKCA, RoHS, RCM, FCC]
+certifications: [CE, UKCA, RoHS, RCM]
```

Comment:
@Matteo-it this is not matching the main branch - it should contain the FCC certification

Reply:
We just need to make sure your branch is updated to main before merging ⭐